packages/kit/src/exports/vite/dev/index.js

```diff
@@ -175,15 +175,20 @@ export async function dev(vite, vite_config, svelte_config) {
 							const styles = {};
 
 							for (const dep of deps) {
-								const parsed = new URL(dep.url, 'http://localhost/');
-								const query = parsed.searchParams;
+								const url = new URL(dep.url, 'http://localhost/');
+								const query = url.searchParams;
 
 								if (
 									isCSSRequest(dep.file) ||
 									(query.has('svelte') && query.get('type') === 'style')
 								) {
+									// setting `?inline` to load CSS modules as css string
+									query.set('inline', '');
```

Comment:
does this have any effect outside CSS modules? am wondering if we should only do this in the CSS module case (i.e. `dep.file.endsWith('.module.css')`) or if it's safe to do it everywhere

Reply:
I think it's safe to do it everywhere, and in fact importing CSS directly to get the preprocessed CSS string is deprecated in Vite 4, though looks like we missed making a warning in `ssrLoadModule`.

In Vite 4, to get preprocessed CSS string only, you'd need to use `?inline`. Because before, `.css` imports implied two things: (1) inject style to head and (2) get css string. And Vite has to hack around with the `?used` query.